sparkmagic/sparkmagic/livyclientlib/linearretrypolicy.py

```diff
@@ -8,12 +8,12 @@ class LinearRetryPolicy(object):
 
     def __init__(self, seconds_to_sleep, max_retries):
         self._seconds_to_sleep = seconds_to_sleep
-        self._max_retries = max_retries
```

Comment:
what's the difference here? starting with a `_` indicates this is for privately usage?